FHEM/00_SIGNALduino.pm

```diff
@@ -226,7 +226,7 @@ SIGNALduino_Initialize($)
 					  ." hexFile"
                       ." initCommands"
                       ." flashCommand"
-  					  ." hardware:ESP_1M,ESP32,nano328,nanoCC1101,miniculCC1101,promini,radinoCC1101"
+  					." hardware:ESP8266,ESP8266cc1101,ESP32,nano328,nanoCC1101,miniculCC1101,promini,radinoCC1101"
```

Comment:
Das Attribut ESP8266cc1101 würde mir noch auffallen, dass es in den Tests noch nicht berücksichtigt ist.

Was passiert allerdings, wenn ESP_1M noch hinterlegt ist. Kennen wir das Verhalten?

Reply:
> Was passiert allerdings, wenn ESP_1M noch hinterlegt ist. Kennen wir das Verhalten?

Da das ganze nur Auswirkungen auf die Auswahl von der FW hat, sollte es keine Rolle spielen weil man auch nicht via WLAN derzeit flashen kann.

Reply:
Ich habe es ergänzt

Reply:
*thanks*